cryptographic_estimators/LEEstimator/LEAlgorithms/leon.py

```diff
@@ -22,26 +22,23 @@
 
 
 class Leon(PELeon, LEAlgorithm):
-
     def __init__(self, problem: LEProblem, **kwargs):
-        """
-        Complexity estimate of Leons algorithm [Leo82]_
+        """Complexity estimate of Leons algorithm [Leo82]_.
+
         Estimates are adapted versions of the scripts derived in [Beu20]_ with the code accessible at
         https://github.com/WardBeullens/LESS_Attack
 
-        INPUT:
+        Args:
+            problem (LEProblem): PEProblem object including all necessary parameters
+            **kwargs: Additional keyword arguments
+                codewords_needed_for_success (int): Number of low word codewords needed for success (default = 100)
+                sd_parameters (dict): Dictionary of parameters for SDFqEstimator used as a subroutine (default: {})
```

Comment:
wrong indentation

Reply:
To be fixed in #198 .

Reply:
Nothing to fix. Indentation is correctly parsed by our custom Sphinx extension (see https://github.com/Crypto-TII/CryptographicEstimators/pull/211).